### ISSUE TITLE
Hotkeys for paste from primary buffer and scroll terminal by line

### ISSUE THREAD

**Issue by damanis** [Eugeny/tabby#8695]
**Describe the solution you'd like**
Just add hotkeys:
- paste from primary buffer in addition to middle mouse click
- scroll terminal by line (now by page and top/bottom)

**Comment by retrobit** [comment]
Agreed with the scroll-by-line hotkey feature. Scroll-by-page is supported, but this can be disorienting when needing to fine-tune viewable lines for context.

Currently supported in Windows Terminal using ⇧ + ^ + `up` and ⇧ + ^ + `down`
